windows/386 build failed

### Description

wails version 2.3.1

execute comand: wails build  -debug -o lasvsim-windows-386.exe -platform windows/386

console:
../../../../go/pkg/mod/github.com/wailsapp/wails/v2@v2.3.0/internal/platform/win32/window.go:72:3: cannot use CW_USEDEFAULT (untyped int constant 2147483648) as int value in argument to CreateWindowEx (overflows)

### To Reproduce

wails build  -debug -o lasvsim-windows-386.exe -platform windows/386

### Expected behaviour

buidl success

### Screenshots

_No response_

### Attempted Fixes

_No response_

### System Details

```shell
Wails CLI v2.3.1

Scanning system - Please wait (this may take a long time)...Done.

# System

OS           | MacOS   
Version      | 12.3    
ID           | 21E230  
Go Version   | go1.19.1
Platform     | darwin  
Architecture | arm64   

# Wails

Version | v2.3.1

# Dependencies

Dependency                | Package Name | Status    | Version
Xcode command line tools  | N/A          | Installed | 2395   
npm                       | N/A          | Installed | 8.15.0 
*Xcode                    | N/A          | Available |        
*upx                      | N/A          | Available |        
*nsis                     | N/A          | Installed | v3.08  
* - Optional Dependency

# Diagnosis

Your system is ready for Wails development!
Optional package(s) installation details: 
  - Xcode: Available at https://apps.apple.com/us/app/xcode/id497799835
  - upx : Available at https://upx.github.io/

 ♥   If Wails is useful to you or your company, please consider sponsoring the project:
https://github.com/sponsors/leaanthony
```


### Additional context

_No response_

--- a/v2/internal/platform/win32/consts.go
+++ b/v2/internal/platform/win32/consts.go
@@ -4,10 +4,11 @@ package win32
 
 import (
 	"fmt"
-	"github.com/wailsapp/wails/v2/internal/system/operatingsystem"
-	"golang.org/x/sys/windows"
 	"syscall"
 	"unsafe"
+
+	"github.com/wailsapp/wails/v2/internal/system/operatingsystem"
+	"golang.org/x/sys/windows"
 )
 
 var (
@@ -330,7 +331,7 @@ const (
 	WS_EX_APPWINDOW           = 0x00040000
 	WS_OVERLAPPEDWINDOW       = 0x00000000 | 0x00C00000 | 0x00080000 | 0x00040000 | 0x00020000 | 0x00010000
 	WS_EX_NOREDIRECTIONBITMAP = 0x00200000
-	CW_USEDEFAULT             = 0x80000000
+	CW_USEDEFAULT             = ^0x7fffffff
 
 	NIM_ADD        = 0x00000000
 	NIM_MODIFY     = 0x00000001

